service/src/main/resources/application.yml

```diff
@@ -63,7 +63,10 @@ cbas:
       coreThreadPoolSize: 5
       maxThreadPoolSize: 10
       queueCapacity: 500
-
+    bard:
+      coreThreadPoolSize: 5
+      maxThreadPoolSize: 10
+      queueCapacity: 500
```

Comment:
500 seems fine for run-set recording. If we ever record workflow-by-workflow we'll want this to be much higher.

Do we do anything special if we overflow this capacity?

Reply:
Not at the moment...how is that handled for async workflow submissions? 🤔

Reply:
I don't think we do any special handling when the run set queue overflows. This [stackoverflow answer](https://stackoverflow.com/a/44647863) says that items added to a maxed out queue are rejected (which seems like a good fact to keep in mind, but may not imply the same solution for run set, workflow, or bard logging)

I think solving for this when it comes up again is fine